docs/agent-context/architecture.md

```
@@ -0,0 +1,53 @@
+# Illustra アーキテクチャ
+
+Windows 用画像ビューア。WPF (.NET 9) + MVVM（Prism 9 / DryIoc / イベントアグリゲーター）。単一アプリ + 単一テストプロジェクトの小さめソリューション。
+
+## 主要コンポーネントと関係
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)